README.md

```diff
@@ -248,21 +248,27 @@ defmodule Cookbook.AddIndex do
 end
 ```
 
-### Adding an index concurrently without disabling lock nor transaction
+### Adding an index concurrently without disabling lock or transaction
 
-#### Example
+Concurrently indexes need to set both `@disable_ddl_transaction` and `@disable_migration_lock` to true. [See more](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#index/3-adding-dropping-indexes-concurrently):
```

Comment:
I've already updated `README` but forgot to push it. I'm merging both versions, because you added some nice explanations 👍🏻